source/deployment/advanced-permissions.rst

```diff
@@ -56,17 +56,28 @@ The interface for naming, assigning teams, and editing permissions in a Team Ove
 
 .. image:: ../images/team-scheme.png
 
-Channel Override Permissions (E20)
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Channel moderation (Beta) (E20)
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-*Available in a future release of Enterprise Edition E20*
+*Available in Enterprise Edition E20*
 
-Allows Admins to restrict permissions within specific channels. Permissions under consideration for this phase include:
+From v5.22, the channel moderation settings allow Admins to restrict actions within specific channels. These actions include: 
 
 - **Read-only Channels:** The ability for Admins to turn off posting in specified channels.
+- **Restrict Post Reactions:** Turn off the ability for members and guests to post reactions.
```

Comment:
suggest renaming to simply **Restrict Reactions** - though I'm 2/5 if you prefer to keep this.